renderer/reducers/invoice.js

```diff
@@ -115,14 +115,14 @@ export const createInvoice = (amount, cryptoUnit, memo, isPrivate) => async (
       private: isPrivate || activeWalletSettings.type === 'local',
       expiry: currentConfig.invoices.expire,
     })
-    dispatch(createdInvoice(invoice))
-  } catch (e) {
-    dispatch(invoiceFailed(e))
+    dispatch(createInvoiceSuccess(invoice))
+  } catch (error) {
+    dispatch(createInvoiceFailure(error))
   }
 }
 
 // Receive IPC event for newly created invoice
-export const createdInvoice = invoice => dispatch => {
+export const createInvoiceSuccess = invoice => dispatch => {
   decorateInvoice(invoice)
```

Comment:
Ideally, we shouldn't mutate invoice here (I know it was like this before this PR)

Reply:
Agreed. Have added to my list and will submit a separate PR to fix this.